sol/tests/test_ics23ops.py

```diff
@@ -0,0 +1,62 @@
+from brownie import accounts, Ops_UnitTest
+from brownie.convert import to_bytes, to_string
+import brownie
+import pytest
+import convert
+import base64
+import helpers
+
+
+@pytest.fixture
+def ops():
+    return accounts[0].deploy(Ops_UnitTest)
+
+
+@pytest.mark.parametrize("name,case", helpers.loadJsonTestCases("../testdata/TestLeafOpData.json"))
+def test_leafOp(ops, name, case):
+    if "sha-512" in name:
+        pytest.skip("sha-512 not supported in solidity")
+    opCase = convert.leafOp(case["Op"])
+    key = base64.b64decode(case["Key"] or "")
+    value = base64.b64decode(case["Value"] or "")
+    if case["IsErr"]:
+        with brownie.reverts():
```

Comment:
Does this ensure it throws an exception?

Reply:
[Yes](https://eth-brownie.readthedocs.io/en/stable/api-test.html?highlight=revertcontext#revertcontextmanager)